src/Database/Database.php

```diff
@@ -310,6 +310,57 @@ public function databaseName(): string
 		return $data[0]['db'];
 	}
 
+	/**
+	 * Full-text match on a haystack string that isn't present in the database
+	 *
+	 * @param string $haystack
+	 * @param string $needle Boolean mode search string
+	 * @return bool
+	 * @throws \Exception
+	 */
+	public function performThrowawayFullTextSearch(string $haystack, string $needle): bool
```

Comment:
I still don't really like the name. For me the name should describe the outcome, but not the way it performs the outcome. So this "throwaway" is not need in the function name, I think.

Reply:
It could always happen that at some point in the future we would be able to do this whole stuff in memory, without the need for this table. Then we had to rename the function, although for the user nothing would change.

Reply:
It's true, hence my question about the naming, if not throwaway, then what? "OneOff"?

Reply:
"disposable" ?